src/index.rst

```diff
@@ -92,17 +103,17 @@ Documentation
 Component Level Dependencies
 ============================
 
-Dependencies between components are as follows:
+Axom has the following inter-component dependencies:
 
-- Core has no dependencies, and the other components depend on Core
+- Core has no dependencies and the other components depend on Core
 - Slic optionally depends on Lumberjack
 - Slam, Spin, Primal, Mint, Quest, and Sidre depend on Slic
 - Mint optionally depends on Sidre
 - Quest depends on Slam, Spin, Primal, and Mint
-- Inlet depends on Sidre, and Slic
+- Inlet depends on Sidre and Slic
```

Comment:
Oops this wasn't true to begin with.  Inlet now depends on Primal.

Reply:
Really?!? How does inlet use primal?

Reply:
We expose the Primal Vector class to Lua functions. Super nice functionality from @joshessman-llnl ! 

https://github.com/LLNL/serac/blob/fb6bff50ae0979ed076796daac5171047dd69b79/data/input_files/tests/nonlinear_solid/qs_component_solve.lua#L56-L58

Reply:
Slightly more exciting usage:

https://github.com/LLNL/serac/blob/99d36930650c1d4c75f3a53306ab211a0f60e4b4/data/input_files/tests/thermal_conduction/static_solve_repeated_bcs.lua#L17-L19

Reply:
Even more exciting usage:
https://github.com/LLNL/serac/blob/99d36930650c1d4c75f3a53306ab211a0f60e4b4/data/input_files/tests/nonlinear_solid/dyn_amgx_solve.lua#L69-L79

Reply:
My mind is blown.  I'll change the docs.